.github/workflows/main.yml

```diff
@@ -1,7 +1,6 @@
 name: CI
 
 on:
-  push: # git push
```

Comment:
Push trigger is used to run test on every pushed commit. Sure you want to remove it?

Reply:
Si t'hi fixes veuràs que amb el "on pull_request" ja fa tots els commits:
https://github.com/Som-Energia/openerp_som_addons/blob/main/.github/workflows/integration_config.yml

Reply:
Sorry, my mistake i mixed it with pull_request. push is perfect.